### DISCUSSION TITLE
Abstract, escaping characters and foreign languages

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#2694] [Q&A] [answered]
I am trying to add an abstract to a _quarto.yml file of a foreign language document (a thesis in Portuguese).
Here are some of my difficulties,  I wonder anyone could help me:

1) can't escape inside an abstract:
neither
```yaml
abstract: "Alan's photography..."
```
or 
```yaml
abstract: "Alan\'s photography..."
```
works

2) The thesis I am trying to convert to quarto (from docx) is in Brazilian Portuguese, but even when I change the language in the "preamble" yaml (language: "pt-BR"), I still get words hard coded to english: "Author", "abstract", "Preface"

3) A thesis in Portuguese should have abstracts both in English and Portuguese, I have no idea how to do that.

**Comment by eitsupi** [comment, marked as answer]
You may need to know about yaml.
https://stackoverflow.com/questions/53082932/yaml-docker-compose-spaces-quotes

**Reply by fredguth** [reply]
I just don't understand what happened. Because I opened this discussion after trying:
```
abstract: "Alan's photography...."
```
which produced an error.

According to the answer in stackoverflow that is valid.

I changed the file many times and now, I tried again and it worked.  "sentence with ' inside double quotes".

Thanks anyway.

**Comment by fredguth** [comment]
This solves the abstract error, still don't know the language part of the question.
